src/de/svnr.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * SVNR or RVNR (Sozialversicherungsnummer, German social security / pension insurance number).
+ *
+ * The SVNR (or RVNR) is the German social security / pension insurance number.
+ * It's also called a social insuarance number. The number consists of 12 digits.
+ *
+ * Source
+ *   https://de.wikipedia.org/wiki/Versicherungsnummer
+ *   https://allaboutberlin.com/guides/german-versicherungsnummer
+ *
+ * PERSON
+ */
+
+import * as exceptions from '../exceptions';
+import { strings } from '../util';
+import { weightedSum } from '../util/checksum';
+import { Validator, ValidateReturn } from '../types';
+
+const AREA_NUMBER_OPTIONS = [
+  '02',
+  '03',
+  '04',
+  '08',
+  '09',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '23',
+  '24',
+  '25',
+  '26',
+  '28',
+  '29',
+  '38',
+  '39',
+  '40',
+  '80',
+  '81',
+  '82',
+  '89'
+].concat([...Array(79 - 42 + 1).keys()].map(x => (x + 42).toString())) // '42'-'79'
```

Comment:
In both of these two cases you shouldn't need the `[...` spread operators, since they're returning an array anyways.

Reply:
^ adding `.keys()` returns iterator (`(method) Array<any>.keys(): IterableIterator<number>`), and without spread operator, I'm getting below type error. 
`Operator '+' cannot be applied to types 'IterableIterator<number>' and 'number'.`

Reply:
@koblas Can you approve the pr if no other changes are required?